app/api/activitylog/specs/activitylogMiddleware.spec.js

```diff
@@ -1,19 +1,22 @@
-import { appendFile } from 'api/files';
 import { IGNORED_ENDPOINTS } from 'api/activitylog/activitylogMiddleware';
 import date from 'api/utils/date';
+import asyncFS from 'api/utils/async-fs';
+import { testingTenants } from 'api/utils/testingTenants';
 import activitylogMiddleware from '../activitylogMiddleware';
 import activitylog from '../activitylog';
 
-jest.mock('api/files', () => ({
-  appendFile: jest.fn(),
-  activityLogPath: jest.fn().mockImplementation(() => './log/default_activity.log'),
-}));
-
 describe('activitylogMiddleware', () => {
   let req;
   let res;
   let next;
 
+  beforeAll(() => {
+    testingTenants.mockCurrentTenant({
+      name: 'tenant1',
+      activityLogs: `${__dirname}/../../files/specs/uploads`,
+    });
+  });
+
```

Comment:
@Zasa-san we designed testingEnvironment to do this exactly, but the name is hardcoded there, i suggest you add support for name  use setTenant instead.
```
  beforeAll(() => {
    testingEnvironment.setTenant('tenant1');
  });

```

what do you think ?

Reply:
Yes, I forgot that testingEnviroment already handled file routes. Changed!